pkg/monitor/cluster/clusterflagsandbanner.go

```diff
@@ -0,0 +1,46 @@
+package cluster
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+
+	"github.com/Azure/ARO-RP/pkg/api"
+	arov1alpha1 "github.com/Azure/ARO-RP/pkg/operator/apis/aro.openshift.io/v1alpha1"
+)
+
+const (
+	operatorFlagMetricsTopic  = "cluster.nonstandard.operator.featureflags"
+	supportBannerMetricsTopic = "cluster.nonstandard.contact.support.banner"
+)
+
+func (mon *Monitor) emitOperatorFlagsAndSupportBanner(ctx context.Context) error {
+	cs, err := mon.listAROClusters(ctx)
```

Comment:
see comment above regarding pagination, this now has 2 copies of the entire list of clusters in memory. one in the cache and now a second in this function. this is where we could paginate and perform logic as needed.